lib/awful/mouse/init.lua

```diff
@@ -231,19 +231,16 @@ end
 -- @tparam string context The context
 -- @tparam[opt={}] table hints The hints to pass to the handler
 function mouse.resize_handler(c, context, hints)
-    if hints and context and context:find("mouse.*") then
+    if hints and context and context:find("mouse.*")
+            and not hints.from_mouse_resize_handler then
         -- This handler only handle the floating clients. If the client is tiled,
         -- then it let the layouts handle it.
         local t = c.screen.selected_tag
         local lay = t and t.layout or nil
 
         if (lay and lay == layout.suit.floating) or c.floating then
-            c:geometry {
-                x      = hints.x,
-                y      = hints.y,
-                width  = hints.width,
-                height = hints.height,
-            }
+            hints.from_mouse_resize_handler = true
+            c:emit_signal("request::geometry", context, hints)
```

Comment:
I think you created a race condition. If you allow ewmh.geometry to understand mouse.resize/mouse.move, then it will handle both request::geometry and the size will be applied twice. Better use a different context name to avoid that.